_sass/components/_drawer.scss

```diff
@@ -62,7 +62,10 @@
     display: inline-block;
     float: right;
     margin-bottom: 0;
-    padding: 0;
+    padding: $base-padding-lite;
```

Comment:
Properties should be ordered background-color, color, display, float, margin-bottom, margin-bottom, margin-right, margin-top, padding